File: qe-selenium/qe-selenium-java/src/main/java/com/demos/Activity5.java
package com.demos;

import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity5 {
    public static void main(String[] args) {
        FirefoxDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/webelements/dynamic-controls/");
        System.out.println(driver.getTitle());
        System.out.println(driver.findElement(By.id("textInput")).isEnabled());
        driver.findElement(By.xpath("//*[@id=\"textInputButton\"]")).click();
        System.out.println(driver.findElement(By.id("textInput")).isEnabled());
        driver.quit();

    }
    
}

File: qe-selenium/qe-selenium-python/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By


with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls/")
    print(driver.title)
    print(driver.find_element(By.XPATH,"//*[@id=\"checkbox\"]").is_displayed())
    driver.find_element(By.XPATH,"/html/body/div/main/div/div/div/div/div[2]/section[1]/button").click()
    print(driver.find_element(By.XPATH,"//*[@id=\"checkbox\"]").is_displayed())
    
    driver.quit